src/web/layouts/ShowcaseLayout.tsx

```diff
@@ -128,7 +128,7 @@ export const ShowcaseLayout = ({ CAPI, config, NAV }: Props) => (
         </Section>
 
         <Section islandId="most-viewed">
```

Comment:
This needs to be updated to `most-viewed-grid`

Reply:
Done